Makefile

```diff
@@ -82,7 +82,6 @@ smart_card_connector_app/build: third_party/libusb/naclport/build
 smart_card_connector_app/build: third_party/pcsc-lite/naclport/common/build
 smart_card_connector_app/build: third_party/pcsc-lite/naclport/server/build
 smart_card_connector_app/build: third_party/pcsc-lite/naclport/server_clients_management/build
-third_party/ccid/naclport/build: third_party/libusb/naclport/build
```

Comment:
This dependency is not needed anymore, now that we don't install libusb's headers anymore.